index.html

```diff
@@ -159,7 +165,7 @@ <h5>Skills</h5>
          </ul>
       </div>
    </section>
-   <section class="contact">
+   <section id="contact">
```

Comment:
Since this problem was not raised by the previous reviewer, I will mark this as optional, however, I strongly recommend you implement this, because the way your app is now, doesn't look good.
- Optional
There is a lot of blank space between the footer and contact section. Kindly fix it and also there is a horizontal line on top of the footer, you may consider adding it.
This is how your app is looking now.
![image](https://user-images.githubusercontent.com/33268927/173045540-bbf4ca35-7fd7-4503-a61d-64387faa5d05.png)

Reply:
@muneebulrehman take a look at it onelast time pls